docs/conf.py

```diff
@@ -165,9 +170,15 @@
 
 # A dictionary of values to pass into the template engine’s context for all pages.
 html_context = {
-    'to_be_indexed': ['stable', 'latest']
+    'to_be_indexed': ['stable', 'latest'],
```

Comment:
Addition of trailing comma is unnecessary.
```suggestion
    'to_be_indexed': ['stable', 'latest']
```